tests/acceptance/expected-failures-on-OC-storage.txt

```diff
@@ -534,8 +523,6 @@ apiTrashbin/trashbinRestore.feature:206
 apiTrashbin/trashbinRestore.feature:207
 apiTrashbin/trashbinRestore.feature:279
 apiTrashbin/trashbinRestore.feature:280
-apiTrashbin/trashbinRestore.feature:295
-apiTrashbin/trashbinRestore.feature:310
```

Comment:
These are both Examples for "new" dav-path. The example for "old" dav-path at 294 and 309 were already passing.